README.md

```diff
@@ -675,3 +714,24 @@ Your changes will only take effect in new terminal windows.
 [running_gif]: doc/images/opensim_running.gif
 [simple_example_gif]: doc/images/opensim_double_pendulum_muscle.gif
 [java]: http://www.oracle.com/technetwork/java/javasebusiness/downloads/java-archive-downloads-javase6-419409.html
+
+#### For the impatient (Ubuntu)
+
+* In **Terminal** --
+ * `sudo add-apt-repository ppa:george-edison55/cmake-3.x`
+ * `sudo apt-get update`
+ * `sudo apt-get install git cmake cmake-curses-gui clang-3.6 freeglut3-dev libxi-dev libxmu-dev liblapack-dev`
+ * `sudo rm -f /usr/bin/cc /usr/bin/c++`
+ * `sudo ln -s /usr/bin/clang-3.6 /usr/bin/cc`
+ * `sudo ln -s /usr/bin/clang++-3.6 /usr/bin/c++`
```

Comment:
Another alternative is to define `CC=clang-3.5` and `CXX=clang++-3.6`. I actually think this is the best option, since the user might also have software elsewhere on their computer that they want to compile with gcc, etc.